uiowa/uiowa
# Add status message JS to close dismissable alert.

Resolves #1620 

### Testing
- Check out branch.
- cache:rebuild
- save page or something
- dismiss alert
